RELEASE-NOTES.txt

```diff
@@ -1,3 +1,11 @@
+1.3.5
+=====
+* Improved stats tracking of app usage #988
+* Added failed state when preview sites update fails #977
+* Improved site name suggestions to ensure path uniqueness #972
+* Misc minor bug fixes and UI improvements
+* Fixed updating, export and push for sites with older SQLite database integration #966
```

Comment:
Just a minor idea: We could consider changing the order - as I think the improvement of site name suggestions is more significant to users than improved tracking:

```suggestion
* Improved site name suggestions to ensure path uniqueness #972
* Added failed state when preview sites update fails #977
* Improved stats tracking of app usage #988
* Misc minor bug fixes and UI improvements
* Fixed updating, export and push for sites with older SQLite database integration #966
```

Reply:
We can also move the "misc minor bug" to the end.